tests/function-trace.sh

```diff
@@ -10,7 +10,8 @@ expect_trace() {
             --trace-function-calls \
             --expr "$expr" 2>&1 \
             | grep "function-trace" \
-            | sed -e 's/ [0-9]*$//'
+            | sed -e 's/ [0-9]*$//' \
+            || true
```

Comment:
(nit) Since there's a `set +e` just below, I guess we can just move it a few lines above to make the whole function more consistent

Reply:
Ah I have a way to just get rid of all the `set +e` instead :grinning:

Reply:
Does that mean that you want to keep doing more work on this? Or do you mean as a follow-up?

Reply:
I did as part of this PR (small change), but now I have a seemingly unrelated failure I couldn't reproduce just running `make installcheck`. Frustrating..